docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -59,7 +59,7 @@ it "Get-Item on a nonexisting file should have error PathNotFound" {
     try
     {
         get-item "ThisFileCannotPossiblyExist" -ErrorAction Stop
-        throw "No Exception!"
+        throw "Expected an exception throw, but no Exception was thrown."
```

Comment:
To be grammatically correct, shouldn't it be `an exception throw*n*`?
Personally, I find `We shouldn't get here, the previous command should have thrown an exception.` to be the clearest.